Makefile

```diff
@@ -55,6 +55,7 @@ LDFLAGS := $(TARGET_FLAGS) $(ARCH_FLAGS_${ARCH_CODE})
 BAK_FOLDER := $(shell date +%s)
 PREFIX := /usr/local
 GIT_PREFIX := $(PREFIX)/git
+GIT_BUILT_FROM_COMMIT :=
```

Comment:
Does that really work? I thought that `:=` overrides whatever was specified on the command-line, and we would need `?=` here. But then, any uninitialized variable `VAR` would see `$(VAR)` interpolated to the empty string, no?

Reply:
It does! According to the [documentation](https://www.gnu.org/software/make/manual/html_node/Overriding.html), a command line-defined variable overrides all simply (`:=`) and recursively (`=`) expanded assignments. As a result, this initializes `GIT_BUILT_FROM_COMMIT` as an empty string _only if_ it isn't specified on the command line.